util/argo/audit_logger.go

```diff
@@ -108,6 +108,7 @@ func (l *AuditLogger) LogAppEvent(app *v1alpha1.Application, info EventInfo, mes
 	if user != "" {
 		fields["user"] = user
 	}
+	fields["spec"] = spec.GetProject()
```

Comment:
I think we want the whole spec.

I'd also recommend making a deep copy of the spec and then clearing the `status` field.

```suggestion
	specCopy := spec.DeepCopy()
	specCopy.Status = nil
	fields["spec"] = spec
```

Reply:
Got it, will do!

Reply:
I don't think spec has a Status field from what I can see; also would we want the copy in the map or the original spec?